src/Uno.UI/UI/Xaml/UIElement.macOS.cs

```diff
@@ -109,6 +109,12 @@ internal Windows.Foundation.Point GetPosition(Point position, global::Windows.UI
 		public override bool AcceptsFirstResponder()
 			=> true; // This is required to receive the KeyDown / KeyUp. Note: Key events are then bubble in managed.
 
+
+		internal static void LoadingRootElement(UIElement visualTreeRoot) { }
```

Comment:
Is this a placeholder for future use?

Reply:
Likely. We can keep it as a reminder that it's placed at the right location.

Reply:
Yes @davidjohnoliver indeed, this is some sort of placeholder.